Challenge 5 to 9, 10 , 12, 15,18,19,22

shivanshjaitly opened the pull request.

shivaylamba committed:
Update details

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
Update README.md

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
Make final update to ReadME

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
CHALLENGE 12 DONE!!! SQUASHING FINISHED

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

Create challenge_11.md

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
Add Challenge 1

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
Challenge 5

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
Update challenge_5.md

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

Mrjoy832 reviewed (changes requested):
username-2.md file still exists ...fix it

shivaylamba committed:
Challenge15

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
small update

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
add new challenger

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
add new challenger

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
add challenge 16

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
challenge 19 - 1

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly commented:
I tried to write in the group. No one responded. Now you all gonna send me the message milestone 2 is not achieved. Is it my fault?

shivanshjaitly commented:
![Screenshot (197)](https://user-images.githubusercontent.com/61774851/191823896-2b4f3de0-4946-440c-82d0-ebc5f6a621ed.png)

Mrjoy832 reviewed (changes requested):
## 5 Files changed , Huge no of commits ....fix it .

shivanshjaitly commented:
> ## 5 Files changed , Huge no of commits ....fix it .

MrJoy actually it was giving conflict in challenge 16. It was very difficult to fix it. That's why so many commits are there. Please consider it

shivanshjaitly commented:
> ## 5 Files changed , Huge no of commits ....fix it .

All my challenges are up to date. I haven't skipped or did anything wrong. Linter was failing and challenge 16 was left. After you said fix challenge2.md. I deleted it then DCO was also not passing now after investing 4 hours. It worked man!!!!

Mrjoy832 reviewed (changes requested):
![image](https://user-images.githubusercontent.com/77873383/192073623-d62ef725-82c3-468d-b81e-1a293fadb835.png)
This is extra....No challenge asked you to add this file...remove this file and push again

shivanshjaitly commented:
> ![image](https://user-images.githubusercontent.com/77873383/192073623-d62ef725-82c3-468d-b81e-1a293fadb835.png) This is extra....No challenge asked you to add this file...remove this file and push again

Done MrJoy. Please check and verified

Mrjoy832 commented:
## Milestone-2 Done✅

kuraakhilesh8230 commented:
1.too many commits 
2.not attached screen shots of challenge-22 in pr
complete it asap

shivanshjaitly commented:
> I haven't skipped or did anything wrong. Linter was failing and challenge 16 was left. After you said fix challenge2.md. I deleted it then DCO was also not passing

So, sorry @kuraakhilesh8230  that day network issues were there. Now I have attached the Challenge 22. I had done this on 22nd September. I am attaching the issue link:   https://github.com/scaleracademy/scaler-september-open-source-challenge/issues/75#issuecomment-1259289126

2.  I haven't skipped or did anything wrong. Linter was failing and challenge 16 was left. After @Mrjoy832  said fix challenge2.md. I deleted it then DCO was also not passing. 15-20 times it was giving me error, so I did git rebase again and again just to remove the conflicting. I have also talked about this with MRJOY.

shivanshjaitly commented:
DONE

On Tue, 27 Sept 2022 at 15:13, Kura Akhilesh ***@***.***>
wrote:

> 1.too many commits
> 2.not attached screen shots of challenge-22 in pr
> complete it asap
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/1041#issuecomment-1259252312>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOXJYA3GNIQA62D57AUZM6DWAK6SHANCNFSM6AAAAAAQIZAHTI>
> .
> You are receiving this because you authored the thread.Message ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/1041/c1259252312
> @github.com>
>

kuraakhilesh8230 commented:
Wrong files changed total 35 ??

shivanshjaitly commented:
I am doing it. Give me some minutes man!

On Tue, 27 Sept 2022 at 23:42, Kura Akhilesh ***@***.***>
wrote:

> Wrong files changed total 35 ??
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/1041#issuecomment-1259877183>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOXJYA5JGVGHQZRY3MNHT4TWAM2JLANCNFSM6AAAAAAQIZAHTI>
> .
> You are receiving this because you authored the thread.Message ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/1041/c1259877183
> @github.com>
>

shivanshjaitly committed:
Changes done

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
Changes done

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
Final done

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
...

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly commented:
> Wrong files changed total 35 ??

Now checked all done. Commit reduced. Also working on DCO to pass. Told you before I will do that by tomorrow. I hope you will understand!!!!

shivanshjaitly committed:
DCO trying to pass

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly commented:
DONE!!!!!!! @kuraakhilesh8230    @Mrjoy832   check it now. All done. Lint + DCO passed also commit reduced to 11!!! I hope now it is sorted. Hoping for the positive response

kuraakhilesh8230 commented:
Congratulations you have successfully completed milestone 3 and good to go 👏

shivaylamba committed:
28

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivaylamba committed:
challenge 29

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
Linter issue man, I don't know

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
Update challenge_1.md

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
Problem solved

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
Problem solved

Signed-off-by: shivanshjaitly <shivanshjaitly11@gmail.com>

shivanshjaitly committed:
Merge branch 'main' into shivanshjaitly-details

shivanshjaitly commented:
I did squashing again linter was not passing. What should I do?

On Tue, 27 Sep 2022 at 4:08 PM, Kura Akhilesh ***@***.***>
wrote:

> too many commits this should be reduced
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/scaleracademy/scaler-september-open-source-challenge/pull/1041#issuecomment-1259313540>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOXJYAZAYQGSDNYRB5DORKLWALFCXANCNFSM6AAAAAAQIZAHTI>
> .
> You are receiving this because you authored the thread.Message ID:
> <scaleracademy/scaler-september-open-source-challenge/pull/1041/c1259313540
> @github.com>
>
